lucille-core/src/test/java/com/kmwllc/lucille/core/FileToFileIngestIT.java

```diff
@@ -29,6 +32,7 @@ public void fullIT() throws Exception {
       assertEquals("\"Salad\",\"USA\",\"10\"", lines.get(6));
       assertEquals("\"Wine\",\"France\",\"12\"", lines.get(7));
     } finally {
+      directory.delete();
       f.delete();
```

Comment:
The directory and file delete lines would need to be swapped if we want to delete the directory too